src/transformers/models/mt5/modeling_mt5.py

```diff
@@ -205,11 +205,11 @@ def forward(self, hidden_states):
         return hidden_states
 
 
-# Copied from transformers.models.t5.modeling_t5.T5Attention with T5->MT5
 class MT5Attention(nn.Module):
     def __init__(self, config: MT5Config, has_relative_attention_bias=False):
         super().__init__()
         self.is_decoder = config.is_decoder
+        self.scalable_attention = config.scalable_attention if hasattr(config, "scalable_attention") else False
```

Comment:
Why is it back this way? We don't implement new models re-using existing ones with new config flags that pollute their forward passes. We copy and modify the modeling code as necessary.